go.mod

```diff
@@ -3,11 +3,12 @@ module github.com/lightninglabs/lndclient
 go 1.13
 
 require (
-	github.com/btcsuite/btcd v0.20.1-beta.0.20200730232343-1db1b6f8217f
+	github.com/btcsuite/btcd v0.20.1-beta.0.20200903105316-61634447e719
 	github.com/btcsuite/btclog v0.0.0-20170628155309-84c8d2346e9f
 	github.com/btcsuite/btcutil v1.0.2
 	github.com/btcsuite/btcwallet/wtxmgr v1.2.0
-	github.com/lightningnetwork/lnd v0.11.0-beta
+	//TODO(carla): bump to 0.12 once cut.
+	github.com/lightningnetwork/lnd v0.11.0-beta.rc4.0.20201118235732-1efef51268d6
```

Comment:
Can we drop this whole commit then, if we're merging against master?

Reply:
Not sure whether I'm following this, this change has new fields in the chanacceptor response which will only be included in the 0.12 release? 

> Or should the master branch only contain functionality that is backward compatible to at least the last major version and everything that requires an update should live in a branch?

So by this approach, we leave this PR on the 0.12 branch until it's released.

Reply:
Ah, I wasn't aware that there were new fields added in `lnd` for this to work. I was confused because you remove the `SubscribeHtlcEvents` code and assumed this made it backward compatible :see_no_evil: 

But I think you need to close the PR and reopen against the 0.12 branch then?